to-do-list.css

```diff
@@ -0,0 +1,66 @@
+html,
+body {
+  font-family: 'VT323', monospace;
+  /* font-family: 'Teko', sans-serif; */
+  font-size: 25px;
+  background-color: #0a766f;
+  color: #a6fe01;
+  display: flex;
+  flex-direction: column;
+  margin: 0;
+  padding:0;
+}
+.big-header {
+  padding-left: 15px;
+  padding-bottom: 0;
+  margin: 0;
+}
+
+.label {
+  padding-left: 15px;
+}
+input[type="text"], textarea {
+  padding: 0;
+  background-color: #0ea49a;
+  border-style: dashed;
+  border-radius: 5px;
+  border-width: 1px;
+  border-color: #a6fe01;
+  color: #a6fe01; /* green */
+  /* color: #ff8000; orange */
```

Comment:
Do you need it?

Reply:
Ok. I commented out the part related to border width. However, I think I need the border color and text color specified because it's black by default. Correct me if I'm wrong please.